routes/infrastructure/construct.py

```diff
@@ -0,0 +1,95 @@
+"""CDK Construct for a Cloudfront Distribution."""
+from typing import Optional
+from urllib.parse import urlparse
+
+from aws_cdk import CfnOutput, Duration, Stack
+from aws_cdk import aws_certificatemanager as certificatemanager
+from aws_cdk import aws_cloudfront as cf
+from aws_cdk import aws_cloudfront_origins as origins
+from aws_cdk import aws_s3 as s3
+from constructs import Construct
+
+from .config import veda_route_settings
+
+
+class CloudfrontDistributionConstruct(Construct):
+    """CDK Construct for a Cloudfront Distribution."""
+
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        raster_api_id: str,
+        stac_api_id: str,
+        region: Optional[str],
+        **kwargs,
+    ) -> None:
+        """."""
+        super().__init__(scope, construct_id)
+
+        stack_name = Stack.of(self).stack_name
+
+        s3Bucket = s3.Bucket.from_bucket_name(
+            self,
+            "stac-browser-bucket",
+            bucket_name=veda_route_settings.stac_browser_bucket,
+        )
+
+        no_cache_policy = cf.CachePolicy(
```

Comment:
Can we use the `CachingDisabled` [managed policy](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/using-managed-cache-policies.html#managed-cache-policy-caching-disabled) instead ? Looks pretty similar ?

Longer note : 

So we're turning off any caching Cloud Front is doing with that policy. We've actually been doing that on MAAP as well (albeit manually on the console, so I don't have any code to link), after noticing that STAC metadata wasn't getting updated frequently enough on the STAC browser.

However, I'm still puzzled that I am doing this because, isn't the point of CloudFront to cache the content in the closest edge point to the user and decrease latency thanks to that ? If we turn that off entirely, why are we using CloudFront ? 

In the first place I remember that we started to use CloudFront because that's what the [AWS docs tell us to do](https://docs.aws.amazon.com/AmazonS3/latest/userguide/WebsiteHosting.html) for serving static content stored in S3 over HTTPs. However, is the CloudFront layer really needed for this, can't we just create a route53 record that directly points to the bucket ? 

_Note : those are mainly questions, I am not suggesting we stop using CloudFront_